setup/WDS.nsi

```diff
@@ -267,6 +267,7 @@ Function CreateUninstallEntry
   WriteRegExpandStr HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\WinDirStat" "InstallLocation" "$INSTDIR"
   WriteRegStr HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\WinDirStat" "DisplayName" "${sVersionFull}"
   WriteRegStr HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\WinDirStat" "DisplayIcon" "$INSTDIR\windirstat.exe,0"
+  WriteRegStr HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\WinDirStat" "DisplayVersion" "${dVersionMajor}.${dVersionMinor}.${dVersionRev}"
   WriteRegDWORD HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\WinDirStat" "dwVersionMajor" "${dVersionMajor}"
   WriteRegDWORD HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\WinDirStat" "dwVersionMinor" "${dVersionMinor}"
   WriteRegDWORD HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\WinDirStat" "dwVersionRev" "${dVersionRev}"
```

Comment:
Should also include VersionMajor and VersionMinor:
```nsi
+ WriteRegDWORD HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\WinDirStat" "VersionMajor" "${dVersionMajor}"
+ WriteRegDWORD HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\WinDirStat" "VersionMinor" "${dVersionMinor}"
```
<sup>See [microsoft](https://learn.microsoft.com/en-us/windows/win32/msi/uninstall-registry-key), [nsis](https://nsis.sourceforge.io/Docs/AppendixD.html#useful_add_uninst_infos), [comment](https://github.com/microsoft/winget-cli/issues/1255#issuecomment-902873661).</sup>

Reply:
Can't find other instances of `dwVersionMajor`, `dwVersionMinor`, `dwVersionRev` and `dwVersionBuild` anywhere, they don't appear to be valid for the uninstall reg.
May be easier to remove dw from `dwVersionMajor` and `dwVersionMinor` (only those two, because `VersionRev` and `VersionBuild` aren't valid)
Unless those dw… keys are used somewhere else in the code. I couldn't find it.

Reply:
Maybe those dw keys are from a this kind of struct?
https://learn.microsoft.com/en-us/windows/win32/api/winnt/ns-winnt-osversioninfoa

But I can fix the VersionMajor and VersionMinor parts.

Reply:
Nope, `OSVERSIONINFO` is an entirely different beast, although I'll concede that the naming was probably inspired by it a bit 😉